features/step_definitions/git-flow.js

```diff
@@ -143,16 +159,77 @@ defineSupportCode(({ Before, Given, When, Then }) => {
             });
     });
 
-    Then(/^a new build from `main` should be created to test that change$/, {
+    When(/^a tag is created$/, {
         timeout: TIMEOUT
     }, function step() {
+        const branch = this.branch;
+        const tag = this.tag;
+
+        return github.createBranch(branch, this.repoOrg, this.repoName)
+            .then(() => github.createFile(branch, this.repoOrg, this.repoName))
+            .then(({ data }) => {
+                this.sha = data.commit.sha;
+                github.createTag(tag, branch, this.repoOrg, this.repoName);
```

Comment:
need `return` for returning Promise Object to cucumber?

Reply:
You're right. We need to return Promise Object.